Mathlib/Analysis/Convex/SpecificFunctions/Basic.lean

```diff
@@ -205,6 +205,11 @@ theorem convexOn_rpow {p : ℝ} (hp : 1 ≤ p) : ConvexOn ℝ (Ici 0) fun x : 
   · simpa using convexOn_id (convex_Ici _)
   exact (strictConvexOn_rpow hp).convexOn
 
+theorem convexOn_rpow_left {b : ℝ} (hb : 0 < b) : ConvexOn ℝ Set.univ (fun (x : ℝ) => b ^ x) := by
```

Comment:
Can you also add a version with `Convex`?

Reply:
:facepalm: You're right; that was my reflex about Continuous/DifferentialbleOn kicking in.